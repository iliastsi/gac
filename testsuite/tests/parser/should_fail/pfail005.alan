hello() : proc
  (x = 3);
{

}
