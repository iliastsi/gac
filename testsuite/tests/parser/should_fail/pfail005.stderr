pfail005.alan:2:3: Error: Parse error on input `('
	Expected function or variable declaration

